src/movement.cpp

```diff
@@ -994,16 +994,18 @@ bool MoveEvent::executeStep(Creature* creature, Item* item, const Position& pos)
 
 ReturnValue MoveEvent::fireEquip(Player* player, Item* item, slots_t slot, bool isCheck)
 {
+	ReturnValue ret = RETURNVALUE_NOERROR;
+	if (equipFunction) {
```

Comment:
```C++
if (equipFunction) {
    ret = equipFunction(this, player, item, slot, isCheck);
}

if (scripted && ret == RETURNVALUE_NOERROR && !executeEquip(player, item, slot, isCheck)) {
    ret = RETURNVALUE_CANNOTBEDRESSED;
}
```

Reply:
I thought the same (adding a () around ```ret == RETURNVALUE_NOERROR```) but maybe 3 checks in the same is not aesthetics or debuggable enough...

Reply:
suggestion looks fine, even tho it's 3 checks it's readable